### DISCUSSION TITLE
What version do I need use in order to attend new GA requirements?

### DISCUSSION THREAD

**Discussion by ghost** [invertase/react-native-firebase#5848] [Q&A] [answered]
Hello there,

According to this [GA page](https://support.google.com/analytics/answer/10960488), every app available in Play Store that uses Firebase Google Analytics should be actually using the Google Analytics 4 property implementation until February 15th 2022. 

What version from this package should I be using to match this requirement?

**Comment by mikehardy** [comment, marked as answer]
Hi there! I don't see any thing on the page you linked that references a need to update software at all, it appears to be administrative on the firebase console side.

Have you performed those steps, and were there any issues completing the process? 
I checked my various projects and none of them displayed the ability to begin the upgrade, following their instructions to see that prompt, so I'm unsure personally.

I may only assume that if migration is happening now, current versions make it possible, so my best advice is to use our current versions here without overriding the underlying SDK versions

**Reply by ghost** [reply]
Hi @mikehardy, thank you for coming back so quickly. I saw this instruction when we were doing the administrative effort, but I guess we can ignore it right?
![iOS - step 1](https://user-images.githubusercontent.com/82293995/141477456-b326566c-5d79-49bb-898a-fefc12678cb0.jpeg)
![image](https://user-images.githubusercontent.com/82293995/141477496-50e8ac86-5ea6-4b08-b37a-5e7ef1e3b5d9.png)

**Reply by mikehardy** [reply]
Yeah I saw that when I test-drove it just now looking for information to answer you.

Those dependencies are *exactly* what this module does for you, and at the most current versions (in general), so you may ignore them, or stated differently, you may be confident they are handled for you by this module

https://github.com/invertase/react-native-firebase/blob/d0d9bd84c1ea231437e9129c62ed68ccd3a4a181/packages/analytics/android/build.gradle#L140-L141

https://github.com/invertase/react-native-firebase/blob/d0d9bd84c1ea231437e9129c62ed68ccd3a4a181/packages/analytics/RNFBAnalytics.podspec#L36-L50

Note our BoM is 29.0.0 right now and Pods release is 8.9.1

https://github.com/invertase/react-native-firebase/blob/d0d9bd84c1ea231437e9129c62ed68ccd3a4a181/packages/app/package.json#L66-L75

**Reply by ghost** [reply]
Thank you for the clarification, I'll update the lib to the latest stable version just to be on the safe side. Have a nice day!
